docs/developer/checkout/troubleshooting.mdx

```diff
@@ -11,3 +11,52 @@ sidebar_label: Troubleshooting
 ### Variant/Product is not available
 
 Make sure product is available in the channel the checkout is created. See [Product Troubleshooting](/developer/products/troubleshooting.mdx).
+
+### Resolving Paid Checkouts Not Converting to Orders
+
+Sometimes, a `Checkout` that has been paid might not convert into an `Order`.
+This can happen for various reasons, such as the customer closing the browser before the `checkoutComplete` mutation was called.
+To ensure that fully paid checkouts are reliably converted into orders, consider implementing one of the following solutions:
+
+#### Solution 1: Use a Webhook to Complete Checkouts
+
+Set up the `CHECKOUT_FULLY_PAID` webhook, and create an app to call the `checkoutComplete` mutation when this webhook is triggered.
+	•	For more information on setting up webhooks, refer to the [webhook overview](/developer/extending/webhooks/overview).
+	•	You can find guidance on app development [here](/developer/extending/apps/overview).
+
+	Note: This approach only works when the `CHARGE` is used as the default transaction flow strategy.
+
+#### Solution 2: Enable Automatic Checkout Completion
+
+Enable [automatic checkout completion](/developer/payments/transactions#automatic-checkout-completion)
+to automatically convert a `Checkout` into an `Order` once it is fully paid.
+
+A checkout is considered fully paid when `Checkout.authorizeStatus` is set to `FULL`, meaning the associated `TransactionItems` cover the `checkout.totalPrice`.
```

Comment:
is that true for both transaction flows (authorize and charge)?

Reply:
Yes. When `authorizeStatus` is updated, both `total_authorized` and `total_charged` are taken into consideration.